ApplicationLibCode/ProjectDataModel/Annotations/RimTimeAxisAnnotation.cpp

```diff
@@ -47,22 +47,38 @@ RimTimeAxisAnnotation::RimTimeAxisAnnotation()
 //--------------------------------------------------------------------------------------------------
 ///
 //--------------------------------------------------------------------------------------------------
-void RimTimeAxisAnnotation::setTime( time_t time, const QString& dateTimeFormatString )
+RimTimeAxisAnnotation* RimTimeAxisAnnotation::createTimeAnnotation( time_t              time,
+                                                                    const cvf::Color3f& color,
+                                                                    const QString&      dateTimeFormatString /*= QString() */ )
```

Comment:
No need to have the empty string comment here?

Reply:
Fixed